src/main/resources/edu/cornell/kfs/fp/document/workflow/CU-RecurringDisbursementVoucherDocument.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='UTF-8'?> 
```

Comment:
Do we need the CU- in the file name? That seems inconsistent with the name we've been using everywhere else.